src/main/resources/site/pages/default/default.ts

```diff
@@ -66,7 +66,6 @@ const previewOverride: object = {
   contentList: 'relatedFactPage',
 }
 
-export const GA_TRACKING_ID: string | null = app.config?.GA_TRACKING_ID || null
```

Comment:
Siden vi henter denne fra config betyr det at vi må rydde bort GA_TRACKING_ID i mimir-config-repo også. Må vel gjøres etter at vi har deployet til alle miljøene for å gjøre det enkelt :)